bhttp/src/lib.rs

```diff
@@ -546,12 +597,10 @@ impl ControlData {
     }
 
     #[must_use]
-    fn code(&self, mode: Mode) -> u64 {
-        match (self, mode) {
-            (Self::Request { .. }, Mode::KnownLength) => 0,
-            (Self::Response(_), Mode::KnownLength) => 1,
-            (Self::Request { .. }, Mode::IndeterminateLength) => 2,
-            (Self::Response(_), Mode::IndeterminateLength) => 3,
+    fn framing_indicator(&self, mode: Mode) -> FramingIndicator {
```

Comment:
This seems like a useful addition either way.  Maybe we can land this tweak separately.